xml/security_fips.xml

```diff
@@ -32,124 +30,187 @@
   <para>
-   &sles; &productnumber; is in the process of implementing the
-   FIPS 140-3 standard. The relevant binaries are undergoing certification,
-   and will be updated in the near future.
+   &productname; &productnumber; is in the process of implementing the FIPS
+   140-3 standard. The relevant binaries are undergoing certification, and will
+   be updated in the near future.
   </para>
   <para>
    For further details, contact your &suse; sales representative.
   </para>
  </important>
-
-<sect1 xml:id="sec-fips-concept">
+ <sect1 xml:id="sec-fips-concept">
   <title>FIPS overview</title>
+
   <para>
-    Every vendor that develops and maintains cryptographic applications, and
-    wants them to be tested for FIPS compliance, must submit them to the
-    Cryptographic Module Validation Program (CMVP) (see
-    <link xlink:href="https://csrc.nist.gov/projects/cryptographic-module-validation-program"/>).
+   Every vendor that develops and maintains cryptographic applications, and
+   wants them to be tested for FIPS compliance, must submit them to the
+   Cryptographic Module Validation Program (CMVP) (see
+   <link xlink:href="https://csrc.nist.gov/projects/cryptographic-module-validation-program"/>).
   </para>
+
   <para>
-    The latest FIPS 140-3 standard was approved in March 2019,
-    and is replacing 140-2.
+   The latest FIPS 140-3 standard was approved in March 2019, and is replacing
+   140-2.
   </para>
 
   <!--taroth 2022-10-17: disabling the following link for now because it
    still is based on FIPS 140-2-->
-  <!--<para>
-    &suse; maintains a list of certified modules at
-    <link xlink:href="https://www.suse.com/support/security/certifications/"/>, along with a lot of other useful information.
-  </para>-->
-</sect1>
 
-<sect1 xml:id="sec-fips-use-case">
+  <!--<para>
+   &suse; maintains a list of certified modules at
+   <link xlink:href="https://www.suse.com/support/security/certifications/"/>, along with a lot of other useful information.
+   </para>-->
+ </sect1>
+ <sect1 xml:id="sec-fips-use-case">
   <title>When to enable FIPS mode</title>
 
-    <warning>
-    <title>FIPS requires expertise</title>
-    <para>
-      Administering FIPS is complex and requires significant expertise.
-      Implementing it correctly, testing, and troubleshooting all require
-      a high degree of knowledge.
-    </para>
+  <warning>
+   <title>FIPS requires expertise</title>
+   <para>
+    Administering FIPS is complex and requires significant expertise.
+    Implementing it correctly, testing, and troubleshooting all require a high
+    degree of knowledge.
+   </para>
   </warning>
 
   <para>
-    The use case for enabling FIPS mode on &sle; is simple: run your
-    server in FIPS mode it when it is required to meet compliance rules.
+   Run your &productnameshort; in FIPS mode when it is required to meet
+   compliance rules, otherwise we do not recommend running your systems in FIPS mode.
```

Comment:
```suggestion
   Only run your &productnameshort; in FIPS mode when it is required to meet
   compliance rules. Otherwise we do not recommend running your systems in FIPS mode.
```